modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -11,9 +11,7 @@
 
 :context: calculating-{prod-id-short}-resource-requirements
 
-This section describes how to calculate resources, such as memory and CPU, required to run {prod}.
-
-The {prod-short} Operator, {devworkspace} Controller, and user workspaces consist of a set of pods. Those pods contribute to the resource consumption in terms of CPU and RAM limits and requests.
+The {prod-short} Operator, {devworkspace} Controller, and user workspaces consist of a set of pods. The pods contribute to the resource consumption in terms of CPU and RAM limits and requests. This section describes how to calculate resources, such as memory and CPU, required to run {prod}.
```

Comment:
Last sentence, minor change. IBM SG wants us to avoid self referential language, like `this section describes` or `this guide instructs`. What do you think of this rewrite? 

`Use the set of pods to calculate resources, such as memory and CPU, that are required to run {prod}.`

Reply:
What about:

Learn how to calculate resources, such as memory and CPU, required to run {prod}.

Reply:
Or maybe do away with the sentence completely?

Reply:
I like the rewrite `Learn how to calculate resources, such as memory and CPU, required to run {prod}.` because it motivates the user to keep reading your documentation and sets users up for what they'll do (learn) with this documentation. 

However, I think it's fine to eliminate it all together. Especially considering minimalism principles. 

either way works for me, so let me know which you'll do and I'll give it the lgtm :)